pkgs/by-name/nu/nuv/tests.nix

```diff
@@ -0,0 +1,17 @@
+{ runCommand, nuv, pname, version }:
+
+runCommand "nuv-test-run"
+{
+  nativeBuildInputs = [ nuv ];
+} ''
+  export TMP_BASE=$(mktemp -d /tmp/.nuv-XXXXX)
+  export HOME=$TMP_BASE
+  export NUV_REPO=""
+  export NUV_ROOT=$TMP_BASE/.nuv/3.0.0/olaris
+  rm -rf $TMP_BASE/.nuv && \
```

Comment:
Why is this creation of directories needed?

Reply:
Yes, these are needed because the "nuv" command attempts to download some tasks from another repository during startup. This is by design. To have the program version without network activity, is needed to emulate the presences of that directory.